script/update-docs.rb

```diff
@@ -174,7 +174,8 @@ def index_l10n_doc(filter_tags, doc_list, get_content)
 
       # HTML anchor on hdlist1 (i.e. command options)
       html.gsub!(/<dt class="hdlist1">(.*?)<\/dt>/) do |_m|
-        text = $1.gsub(/<\/?(?:code|placeholder)>/, "").tr("^A-Za-z0-9-", "")
+        dt_content = $1
```

Comment:
`dt_content` is an awful name, let's call it `inner_html` instead.

Also, do _not_ forget to use the variable in lines 186 and 477, which were the entire reason why you had to assign the value of `$1` to a local variable!